website/docs/r/google_service_account_key.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "google"
+page_title: "Google: google_service_accout_key"
+sidebar_current: "docs-google-service-account-key"
+description: |-
+  Allows management of a Google Cloud Platform service account Key Pair
+---
+
+# google\_service\_account\_key
+
+Creates and manages service account key-pairs, which allow the user to establish identity of a service account outside of GCP. For more information, see [the official documentation](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) and [API](https://cloud.google.com/iam/reference/rest/v1/projects.serviceAccounts.keys).
+
+
+## Example Usage, creating a new Key Pair
+
+```hcl
+resource "google_service_account" "acceptance" {
+  account_id = "%v"
+  display_name = "%v"
+}
+
+resource "google_service_account_key" "acceptance" {
+  service_account_id = "${google_service_account.acceptance.id}"
+  public_key_type = "TYPE_X509_PEM_FILE"
+}
+```
+
+## Create new Key Pair, encrypting the private key with a PGP Key
+
+```hcl
+resource "google_service_account" "acceptance" {
+  account_id = "%v"
+  display_name = "%v"
+}
+
+resource "google_service_account_key" "acceptance" {
+  service_account_id = "${google_service_account.acceptance.id}"
+  pgp_key = "keybase:keybaseusername"
+  public_key_type = "TYPE_X509_PEM_FILE"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+* `name` - The name used for this key pair (not used on create)
```

Comment:
Since `name` is computed, it probably belongs in the `Attributes Reference` section (and then you can get rid of the parenthetical)